.github/workflows/sidecar-build.yaml

```diff
@@ -10,16 +10,11 @@ defaults:
   run:
     working-directory: ./
 jobs:
-  deploy:
-    if: github.repository_owner == 'bcgov'
-    runs-on: ubuntu-latest
-    steps:
-      - name: Image Build for Sidecar utility Image
-        uses: redhat-developer/openshift-actions@v1.1
-        with:
-          version: "latest"
-          openshift_server_url: ${{ secrets.OpenShiftServerURL }}
-          parameters: '{"apitoken": "${{ secrets.OpenShiftToken }}", "acceptUntrustedCerts": "true"}'
-          cmd: |
-            oc project 23576a-tools
-            oc start-build wordpress-sidecar-run --follow
\ No newline at end of file
+  oc:
+    uses: bcgov/wordpress/.github/workflows/oc-setup.yaml@main
+    with:
+      image: wordpress-sidecar-run
+    secrets:
+      openshift_server_url: ${{ secrets.OpenShiftServerURL }}
```

Comment:
I wonder if this `secrets:...` section could also be consolidated in its own .yaml file and used just like 
`uses:bcgov/wordpress.github/workflows/oc-setup.yaml@main`

Reply:
I admit it's a little strange to be repeating the secrets section after doing all the effort of making everything else reusable, but I think this may be a limitation of the `use:` keyword. I had trouble getting the reusable action to recognize secrets which were not explicitly passed into it.

Worth playing around with in the future as I agree that there must be a better way, but I'm going to call this MVP for the time being.

Here are the docs, if you're interested: https://docs.github.com/en/actions/using-workflows/reusing-workflows